client/src/components/pages/Landing.js

```diff
@@ -1,16 +1,21 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 class Landing extends Component {
-  render() {
-    return (
-      <div>
-        <h4>Welcome to Poodl!</h4>
-        <p>The member management tool for rural senior centers!</p>
-        <p>Users please see our <a href="https://github.com/dog-house-development/poodl/wiki/User-Documentation">User Documentation.</a></p>
-        <p>Developers please see our <a href="https://github.com/dog-house-development/poodl/">GitHub Repository.</a></p>
-      </div>
-    );
-  }
+    render() {
+        return (
+            <div className="landing-container">
+                <div className="landing-header">
+                    <h1>Welcome to,</h1>
+                    <h1 className="landing-title">Poodl</h1>
+                    <p>The member management tool for rural senior centers!</p>
+                    <Link to="/" className="button large primary">
+                        Learn more
```

Comment:
Why do you have this button if it goes to the same page?

Reply:
It is just something that I put there for now, we don't really have much that we can put on out landing page.